code/game/objects/structures/kitchen_spike.dm

```diff
@@ -2,13 +2,36 @@
 #define SKINTYPE_MONKEY 1
 #define SKINTYPE_ALIEN 2
 #define SKINTYPE_BEAR 3
+#define SKINTYPE_CORGI 4
 
 #define MEATTYPE_MONKEY 1
 #define MEATTYPE_ALIEN 2
 #define MEATTYPE_BEAR 3
+#define MEATTYPE_CORGI 4
 
 //////Kitchen Spike
 
+/obj/structure/kitchenspike_frame
+	name = "meatspike frame"
+	icon = 'icons/obj/kitchen.dmi'
+	icon_state = "spikeframe"
+	desc = "The frame of a meat spike."
+	density = 1
+	anchored = 0
+
+/obj/structure/kitchenspike_frame/attackby(obj/item/I, mob/user, params)
+	add_fingerprint(user)
+	if(default_unfasten_wrench(user, I))
+		return
+	else if(istype(I, /obj/item/stack/rods))
+		var/obj/item/stack/rods/R = I
+		if(R.get_amount() >= 4)
+			R.use(4)
+			user << "<span class='notice'>You add spikes to the frame.</span>"
+			var/obj/F = new /obj/structure/kitchenspike(src.loc,)

```

Comment:
you know

this isnt for this PR but

standardizing the "make shit, add fingerprints, qdel old shit" into a proc might be nice

you dont have to do it here tho